pycpp/plugins.py

```diff
@@ -52,13 +55,27 @@ def visit_named_temp_file(self, node, vargs):
         node.result_type = True
         return "NamedTempFile::new()"
 
+    def _translate_file(file: str) -> str:
+        FILE_MAP: Dict[object, str] = {
+            sys.stdout: "std::cout",
+            sys.stdin: "std::cin",
+            sys.stderr: "std::cerr",
+        }
+        return FILE_MAP.get(class_for_typename(file, file), file)
```

Comment:
Should class_for_Typename return sys.stdout/stdin/stderr or io.TextIOWrapper here?

Reply:
@adsharma @jayvdb

Reply:
IMO they should be `io.TextIOWrapper`.

We already have plugins for `io.TextIOWrapper.read` and `io.TextIOWrapper.write`

For this PR a new test case is needed in tests/cases/ - maybe called `stdio.py`.  The test framework cant handle stdin, but a test case containing `sys.stdout.write("foobar\n")` should suffice to validate that the changes in this PR are working and cant regress.